images/win/Windows2016-Azure.json

```diff
@@ -875,8 +876,11 @@
         },
         {
             "type": "powershell",
+            "environment_vars": [
+                "RUN_SCAN_ANTIVIRUS={{user 'run_scan_antivirus' }}"
```

Comment:
It should backtick ` instead of a single quote.

Reply:
changed to ` instead of '